VimeoUpload/Upload/Networking/Old Upload/VimeoSessionManager+OldUpload.swift

```diff
@@ -117,7 +117,7 @@ extension VimeoSessionManager
     {
         let request = try (self.requestSerializer as! VimeoRequestSerializer).uploadVideoRequestWithSource(source, destination: destination)
         
-        let task = self.uploadTaskWithRequest(request, fromFile: source, progress: progress, completionHandler: { [weak self] (response, responseObject, error) -> Void in
+        let task = self.uploadTaskWithRequest(request, fromFile: source, progress: {(progress) -> Void in }, completionHandler: { [weak self] (response, responseObject, error) -> Void in
```

Comment:
As i understand in AF2 this is a return variable:

- (NSURLSessionUploadTask *)uploadTaskWithRequest:(NSURLRequest *)request
                                         fromFile:(NSURL *)fileURL
                                         progress:(NSProgress * __nullable __autoreleasing * __nullable)progress
                                completionHandler:(nullable void (^)(NSURLResponse *response, id __nullable responseObject, NSError  * __nullable error))completionHandler;

In AF3 - it is now a block var:

- (NSURLSessionUploadTask *)uploadTaskWithRequest:(NSURLRequest *)request
                                         fromFile:(NSURL *)fileURL
                                         progress:(nullable void (^)(NSProgress *uploadProgress)) uploadProgressBlock
                                completionHandler:(nullable void (^)(NSURLResponse *response, id _Nullable responseObject, NSError  * _Nullable error))completionHandler;